apps/wallet/src/ui/app/components/address-input/index.tsx

```diff
@@ -83,10 +85,13 @@ function AddressInput<FormValues>({
 
             <ErrorMessage className={st.error} name="to" component="div" />
 
-            {isValid && formattedValue !== '' && dirty && (
-                <div className={st.validAddress}>
-                    <Icon icon={SuiIcons.Checkmark} className={st.checkmark} />
-                    Valid address
+            {!addressError && touched && (
```

Comment:
```suggestion
            {addressError && touched && (
```

Reply:
this is for a valid address notification